src/main/webapp/app/overview/exercise-details/request-feedback-button/request-feedback-button.component.ts

```
@@ -0,0 +1,124 @@
+        this.isExamExercise = isExamExercise(this.exercise());
+        if (this.isExamExercise || !this.exercise().id) {
+            return;
+        }
+        this.updateParticipation();
+    }
+
+    private updateParticipation() {
+        if (this.exercise().id) {
+            this.exerciseService.getExerciseDetails(this.exercise().id!).subscribe({
+                next: (exerciseResponse: HttpResponse<ExerciseDetailsType>) => {
+                    this.participation = this.participationService.getSpecificStudentParticipation(exerciseResponse.body!.exercise.studentParticipations ?? [], false);
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)